TESTING.md

```diff
@@ -16,7 +16,15 @@ Start a local server:
 ```
 npm start
 ```
-Then open Cypress:
+
+Generate configs locally (copy automation SAs to `cypress/fixtures`)
+
+Start Cypress with SA environment variables:
 ```
-npm cypress:open
+CYPRESS_ADMIN=$(cat cypress/fixtures/duos-automation-admin.json) \
```

Comment:
Can we add the env variables to the cypress:open command defined in `package.json`? This way devs can run `npm run cypress:open` instead of copying this line every time.

Reply:
Yes, I can look at that. I'd like to have a script that pulls these from vault as well so will dig into that as well.